Modules/System/Barcode Encoder/src/Barcode Parameters.Table.al

```diff
@@ -0,0 +1,150 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+table 9200 BarcodeParameters
+{
+    TableType = Temporary;
+    DataCaptionFields = Provider, Symbology, "Input String";
+    DataClassification = CustomerContent;
+
+    fields
+    {
+        field(1; PrimaryKey; Guid)
+        {
+            Caption = 'Primary Key';
+            DataClassification = CustomerContent;
+        }
+        field(5; "Input String"; Text[250])
+        {
+            Caption = 'Input String';
+            DataClassification = CustomerContent;
+        }
+        field(10; Provider; Enum BarcodeProviders)
```

Comment:
The provider should not be in the argument table. Let's have a design discussion before we proceed.